includes/Modules/Search_Console.php

```diff
@@ -589,4 +591,17 @@ protected function setup_assets() {
 			),
 		);
 	}
+
+	/**
+	 * Gets an owner ID for the module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int Owner ID.
+	 */
+	public function get_owner_id() {
+		$owner = new Owner_ID( $this->options );
+		return $owner->get();
+	}
```

Comment:
👍  Since Search Console doesn't have a specific owner, makes sense to rely on the global "owner" here.